src/main/java/com/ebay/sonar/test/github/TestGithub.java

```diff
@@ -4,6 +4,15 @@
 
 	public void testJava(){
 		System.out.println("test");
+		 if ( 3 != 3 ) { // always false
+  	System.out.println("test");
+}
+if ( 3 == 3 && 3 == 3 ) { // if the first one is true, the second one is too
+ 	System.out.println("test");
+}
+if ( 3== 3 || 3 == 3) { // if the first one is true, the second one is too
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "||" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1764)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "==" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1764)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Identical sub-expressions on both sides of operator "==" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1764)